omero_tagsearch/templates/omero_tagsearch/tagnav.html

```diff
@@ -223,6 +229,12 @@
           <h2>{% trans "Tag Search" %}</h2>
           {# {{ tagnav_form.as_p }} #}
           {{ tagnav_form.non_field_errors }}
+
+          <div id="id_userName" class="tagnav-user">
+            <div class="tagnav-icon">&nbsp;</div>
+            &nbsp;<span>{{ user_name }}</span>
+          </div>
```

Comment:
Minor point: An alternative to using JavaScript to hide this block if there's no user is to use Django which keeps things a bit simpler (not a blocker)

```
{% if user_name %}
    <div...
    </div>
{% endif %}
```

Reply:
Thanks!